config/rbac/role.yaml

```
@@ -46,3 +46,29 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - ephemeral-access.argoproj-labs.io
+  resources:
+  - roletemplates
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

I think the manager should only have read access on the role template. Their lifecycle should be managed by another system. It requires update on the finalizer/status only. 

```suggestion
  - get
  - list
  - watch
```